opensearch/src/main/java/org/opensearch/sql/opensearch/monitor/GCedMemoryUsage.java

```diff
@@ -55,13 +56,24 @@ public void setUsage(long value) {
     usage.set(value);
   }
 
+  public static boolean initialized() {
+    return initialized;
+  }
+
   private void registerGCListener() {
-    boolean registered = false;
     List<GarbageCollectorMXBean> gcBeans = ManagementFactory.getGarbageCollectorMXBeans();
+    if (gcBeans.stream()
```

Comment:
Did we get runtime exception on 2.19-dev branch? Which JDK version this feature depend on?

Reply:
Haven't backported to 2.19-dev. I am still working on backporting https://github.com/opensearch-project/sql/pull/4105. Then the JDK version < 20, there is no specific `G1 Concurrent GC` Monitoring registered, ref https://github.com/openjdk/jdk/blob/jdk-19-ga/src/hotspot/share/gc/g1/g1MonitoringSupport.cpp#L91. From JDK 20, `G1 Concurrent GC` was registered in Monitoring, ref https://github.com/openjdk/jdk/blob/jdk-20-ga/src/hotspot/share/gc/g1/g1MonitoringSupport.cpp#L93.

This PR is to verify if there a Concurrent MXBean registered in JDK before initialing a GCedMemoryUsage.